app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -96,4 +96,6 @@ class BloodPressureRepository @Inject constructor(
         .newest100ForPatient(patientUuid)
         .toObservable()
   }
+
+  fun findOne(uuid: UUID): Single<BloodPressureMeasurement> = Single.fromCallable { dao.getOne(uuid) }
```

Comment:
We generally avoid using prefixes like `get` or `find` for these functions. For instance, `measurementsWithSyncStatus()` instead of `findMeasurementsWithSyncStatus()`.

Let's use `measurement(UUID)`?